RecoPixelVertexing/PixelVertexFinding/test/BuildFile.xml

```diff
@@ -2,8 +2,41 @@
 <use name="root"/>
 <use name="FWCore/Framework"/>
 <use name="FWCore/ParameterSet"/>
-<use name="MagneticField/Records"/>
+<use name="HeterogeneousCore/CUDAUtilities"/>
 <use name="MagneticField/Engine"/>
-<use name="TrackingTools/TransientTrack"/>
+<use name="MagneticField/Records"/>
 <use name="RecoVertex/KalmanVertexFit"/>
 <use name="SimDataFormats/Track"/>
+<use name="TrackingTools/TransientTrack"/>
+
+<bin file="gpuVertexFinder_t.cu" name="gpuVertexFinderOneKernel_t">
+  <use name="cuda"/>
+  <flags CUDA_FLAGS="-g -DGPU_DEBUG -DONE_KERNEL"/>
```

Comment:
is it intended to leave the debugging flags as they are?

Reply:
definetively. Otherwise assert will not be triggered!